app/DoctrineMigrations/Version20180502134816.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180502134816 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE `service` CHANGE COLUMN `guid` `guid` CHAR(36) CHARACTER SET \'utf8\' NULL COMMENT \'(DC2Type:guid)\' ;
+');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE `service` CHANGE COLUMN `guid` `guid` CHAR(36) CHARACTER SET \'utf8\' NOT NULL COMMENT \'(DC2Type:guid)\' ;
```

Comment:
En, voor de volledigheid, zal je hier ook iets met de NULL values reeds in de DB moeten doen lijkt me?

Reply:
:+1: